test/healthcheck_test.go

```diff
@@ -935,7 +980,15 @@ func testHealthCheckSuccess(t *testing.T, e env) {
 // TestHealthCheckFailure invokes the unary Check() RPC on the health server
 // with an expired context and expects the RPC to fail.
 func (s) TestHealthCheckFailure(t *testing.T) {
-	for _, e := range listTestEnv() {
+	envs := listTestEnv()
```

Comment:
This old test framework with its environments and a gazillion options is a pain. Do we really have to test this with different environments etc?

Reply:
Testing against all the envs doesn't really provide too much additional coverage. I've changed the test to use a single environment.